sdk/storage/azure-storage-blobs/src/blob_container_client.cpp

```diff
@@ -21,6 +21,78 @@
 
 namespace Azure { namespace Storage { namespace Blobs {
 
+  namespace {
+    Models::BlobItem BlobItemConversion(Models::_detail::BlobItem& item)
+    {
```

Comment:
Please make sure no properties missed here. 
I didn't see meta handling, I think the listing operation should also be able to return meta info?

Reply:
Metadata (and a lot of other properties) are in `Details` field.